### ISSUE TITLE
Assets Failing to Load (on mobile possibly?)

### ISSUE THREAD

**Issue by griffpatch** [LLK/scratch-vm#2053]
### Actual Behavior

My recept scratch project 'Generic Red' (https://scratch.mit.edu/projects/292297781/) has had a surprising number of commentators saying that the game would start with missing platforms (equating to missing costumes assets I believe?).

The game works well on mobile, so a lot more mobile users have been using it, and it is often mentioned by people saying they were using mobile, so perhaps this is linked - also more often than not ipads?

I don't have access to an ipad or iphone to test these things, but I just wanted to raise this incase the project could help diagnose any issues that were not currently explained.

Users will report that either level 1 or level 2 did not load and that there were no platforms (this occurs if the assets fail to load). Upon asking them to reload or them coming back another day, they may comment thanking me for fixing it - but obviously I don't do anything, the assets just happen to load that time is my suspicion?

I do get comments saying that the game doesn't work on mobile devices at all - but I can't tell if that is the case for their device, or just a reflection of what I have described above.

Thanks,

Griffpatch

### Operating System and Browser

Reports generally are tied to mobile, with a bias towards apple devices I believe (cause vs effect unknown)

**Comment by thisandagain** [comment]
@kchadha @paulkaplan We have seen this a bit in testing with mobile Safari as well, but we still don't quite understand the root cause. Let's discuss.

**Comment by BryceLTaylor** [comment]
I can confirm that this is happening on both ios Safari and ios Chrome.  It's ALSO happening on mac Safari.  The first time you load it, the platforms don't appear, but if you reload the page they WILL appear.  The text on the main menu items also frequently don't appear.

I've seen this kind of issue before.  It might be the same as this one: https://github.com/LLK/scratch-vm/issues/1663.    Also, this one: https://github.com/LLK/scratch-gui/issues/3398 Here is another "sprites don't load" issue: https://github.com/LLK/scratch-gui/issues/3297

**Comment by alexwenbj** [comment]
Yes, this happened on iOS.I think maybe the assets number is too large,or assets file size is too large?

**Comment by alexwenbj** [comment]
Maybe this is not a loading problem,it is a render problem.I can see all the assets has been loaded,but can't see in stage.

**Comment by Ryuurock** [comment]
When will you fix this problem please? @thisandagain

**Comment by Ryuurock** [comment]
When will you fix this problem please? @thisandagain

**Comment by Ryuurock** [comment]
Consider using canvg? https://github.com/canvg/canvg After my test, using this library to render svg will be no problem.
